packages/Panel/src/Panel.js

```diff
@@ -250,18 +254,27 @@ const propTypes = {
   slideFrom: PropTypes.oneOf([types.slideFroms.RIGHT, types.slideFroms.LEFT, types.slideFroms.BOTTOM]),
 
   /** The width of the open Panel (when slide in from left or right) */
-  width: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  width: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+    types.widthTypes.SMALL,
+    types.widthTypes.MEDIUM,
+    types.widthTypes.LARGE,
+  ]),
 
   /** Control the z-index of the Panel */
   zIndex: PropTypes.number,
+
+  /** Differentiate between similar coloured UI elements */
+  hasAccent: PropTypes.bool,
 };
 
 const defaultProps = {
   a11yText: null,
   getPushContentElement: null,
   groupOffsetY: 0,
   height: "33%",
-  isCompact: false,
+  size: types.sizes.MEDIUM,
   isInline: false,
   isOpen: false,
   kind: types.kinds.DEFAULT,
```

Comment:
Do we need the `kind` here and on L238-239? 
Please neglect my series of "is this needed comment?" if we actually need them. But if we don't, we might need to edit out in places like Dialog.js and Dialog.styles.js as well